{"uuid": "efa90e91-d41b-4eb9-8b7a-84fe75887f58", "counter": 31, "stdout": "\u001b[0;31mfatal: [demo02]: FAILED! => {\"msg\": \"The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'size'. 'dict object' has no attribute 'size'\\n\\nThe error appears to be in '/tmp/tmpni7c4g_c.yml': line 75, column 7, but may\\nbe elsewhere in the file depending on the exact syntax problem.\\n\\nThe offending line appears to be:\\n\\n\\n    - name: Mostrar resumen ejecutivo\\n      ^ here\\n\"}\u001b[0m", "start_line": 29, "end_line": 30, "runner_ident": "8817b4b7-36a0-42dc-a499-3bc1048bb326", "event": "runner_on_failed", "pid": 169262, "created": "2025-05-17T11:36:27.669738+00:00", "parent_uuid": "005056b7-f67d-c697-8d5d-000000000006", "event_data": {"playbook": "/tmp/tmpni7c4g_c.yml", "playbook_uuid": "39b8f9b8-c97f-4c4a-9974-8cb05af8c29f", "play": "Revisar logs del sistema y autenticaci\u00f3n", "play_uuid": "005056b7-f67d-c697-8d5d-000000000002", "play_pattern": "demo02", "task": "Mostrar resumen ejecutivo", "task_uuid": "005056b7-f67d-c697-8d5d-000000000006", "task_action": "debug", "resolved_action": "ansible.builtin.debug", "task_args": "", "task_path": "/tmp/tmpni7c4g_c.yml:75", "host": "demo02", "remote_addr": "demo02", "res": {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'size'. 'dict object' has no attribute 'size'\n\nThe error appears to be in '/tmp/tmpni7c4g_c.yml': line 75, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Mostrar resumen ejecutivo\n      ^ here\n", "_ansible_no_log": false}, "start": "2025-05-17T11:36:27.609826+00:00", "end": "2025-05-17T11:36:27.669266+00:00", "duration": 0.05944, "ignore_errors": null, "event_loop": null, "uuid": "efa90e91-d41b-4eb9-8b7a-84fe75887f58"}}